app/views/comment_mailer/_comment_notification_footer.html.erb

```diff
@@ -1,7 +1,7 @@
 <p>
-  <%= ts("Posted") %>: <%= @comment.created_at %>
+  <%= t(".posted", datetime: l(@comment.created_at)) %>
```

Comment:
Note: the localization of the date/time in the footer caused a change in the date format in English

Instead of `2025-08-19 12:34:56 UTC`, it's now `Tue, 19 Aug 2025 12:34:56 +0000`
An alternative is `August 19, 2025 12:34` (:long) or a custom format can be defined

Reply:
This format is good!